lib/debug/CMakeLists.txt

```diff
@@ -1,15 +1,16 @@
 set(target hyped_debug)
-
-file(GLOB headers "${CMAKE_CURRENT_SOURCE_DIR}/*.hpp")
-file(GLOB code "${CMAKE_CURRENT_SOURCE_DIR}/*.cpp")
+file(GLOB_RECURSE headers "${CMAKE_CURRENT_SOURCE_DIR}/*.hpp")
+file(GLOB_RECURSE code "${CMAKE_CURRENT_SOURCE_DIR}/*.cpp")
 add_library(${target} STATIC ${headers} ${code})
 include_directories(${target}
   "${CMAKE_SOURCE_DIR}/lib"
   "${CMAKE_CURRENT_SOURCE_DIR}"
   ${rapidjson_SOURCE_DIR}/include
 )
 target_link_libraries(${target} 
+  hyped_core
   hyped_io
   hyped_sensors
   hyped_motors
+  Curses
```

Comment:
Build worked for me by changing `Curses` to `${CURSES_LIBRARIES}`

Reply:
This fixes it for me also